src/Sagas.ts

```diff
@@ -42,7 +42,7 @@ function* signIn(action: Types.IMsalSignInAction): SagaIterator {
     const scopes: string[] = action.scopes || [userAgentApplication.clientId];
 
     if (userAgentApplication.isCallback(window.location.hash)) {
-        userAgentApplication._processCallBack(window.location.hash);
+        // Already handled in userAgentApplication constructor
```

Comment:
Without the context of the PR, this could provide some confusion. Would it be better off removed?

Have you also tested that this works in an application? I had to add it in last time because it didn't process the callback without the manual `_processCallBack()` call.

Reply:
Oh actually I've tested it with new Msal library. The behavior of Msal's userAgentApplication is changed, the `_processCallback()` is renamed to `processCallback()`. I think we can keep it for reference if the library behavior changes again. See https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/9c11975e8dccb34a022ffbc0334f1fb8844930fa/lib/msal-core/src/UserAgentApplication.ts#L289